2022/20xxx/CVE-2022-20698.json

```diff
@@ -1,18 +1,104 @@
 {
```

Comment:
'value' is a required property

Reply:
Fixed in https://github.com/CVEProject/cvelist/pull/4053/commits/27d5b35156045e6253795982d52f4ee296a1d45d